README.md

```diff
@@ -4,29 +4,25 @@
 # BCF REST API
 ![](https://raw.githubusercontent.com/BuildingSMART/BCF/master/Icons/BCFicon128.png)
 
-**Version 2.1** based on BCFv2.1.
-[BCF XML GitHub repository](https://github.com/BuildingSMART/BCF)
+**BCFv2.2** is based on BCFv2.1.
+
+[BCF XML GitHub repository](https://github.com/BuildingSMART/BCF-XML)
+
+[Version 1.0 of the Open CDE Foundation API specification is available here](https://github.com/BuildingSMART/shared-common-API/tree/v1.0)
 
 [Version 1.0 of the BCF API specification is available here](https://github.com/BuildingSMART/BCF-API/tree/v1.0)
 
+[Version 2.1 of the BCF API specification is available here](https://github.com/BuildingSMART/BCF-API/tree/v2.1)
+
 **Table of Contents**
 
 <!-- toc -->
 
 - [1. Introduction](#1-introduction)
-  * [1.1 Paging, Sorting and Filtering](#11-paging-sorting-and-filtering)
-  * [1.2 Caching](#12-caching)
-  * [1.3 Updating Resources via HTTP PUT](#13-updating-resources-via-http-put)
-  * [1.4 Cross Origin Resource Sharing (CORS)](#14-cross-origin-resource-sharing-cors)
-  * [1.5 Http Status Codes](#15-http-status-codes)
-  * [1.6 Error Response Body Format](#16-error-response-body-format)
-  * [1.7 DateTime Format](#17-datetime-format)
-  * [1.8 Authorization](#18-authorization)
-    + [1.8.1 Per-Entity Authorization](#181-per-entity-authorization)
-    + [1.8.2 Determining Authorized Entity Actions](#182-determining-authorized-entity-actions)
-  * [1.9 Additional Response and Request Object Properties](#19-additional-response-and-request-object-properties)
-  * [1.10 Binary File Uploads](#110-binary-file-uploads)
-  * [1.11 Differences Between null and Empty Lists](#111-differences-between-null-and-empty-lists)
```

Comment:
Also the whole section #3 should be removed here in the index

Reply:
Thanks, a huge miss.